worker-batch-ingestion-plugin/pom.xml

```diff
@@ -45,7 +45,7 @@
             <dependency>
                 <groupId>com.github.cafdataprocessing</groupId>
                 <artifactId>worker-document-shared</artifactId>
-                <version>5.0.0-924</version>
+                <version>5.2.0-US616155-SNAPSHOT</version>
```

Comment:
@dermot-hardy I want to update the Ingestion Batch Worker to the latest version of the worker framework and worker document. 

I'm not sure if this is the right way to do it? Since when I went into a running container, I see that it has some older versions of the framework jars, like worker-api-5.1.0-919:

![Capture](https://user-images.githubusercontent.com/55391479/219679984-af320f58-db46-42d4-9368-ab92ef3b3497.PNG)

These are being pulled in by the worker-batch framework:

https://github.com/CAFDataProcessing/staging-service/blob/develop/pom.xml#L60

Or should I update the worker-batch framework first, like I have done [here](https://github.com/JobService/worker-batch/pull/33), and use that, which gives me:

![image](https://user-images.githubusercontent.com/55391479/219681218-95a13b97-3b9a-4eff-8ecd-e95dfc964722.png)

Reply:
Yes I agree it's too complicated.  We will simplify it but yes this is fine.